onnxruntime/core/providers/mkldnn/mkldnn_execution_provider.h

```diff
@@ -147,6 +147,8 @@ class MKLDNNExecutionProvider : public IExecutionProvider {
   }
 
  private:
+  mutable int subgraph_index_ = 0;
```

Comment:
why mutable?

Reply:
We've to increment this variable for every subgraph in GetCapability() and it a const function.

Reply:
Ah yes. CreateMetaDef is const. Missed that.